src/Engine/ProtoCore/Core.cs

```diff
@@ -770,7 +770,11 @@ public void GenerateExecutable()
                 BfsBuildInstructionStreams(CodeBlockList[n], DSExecutable.instrStreamList);
             }
 
-            GenerateExprExe();
+            if (Options.RunMode == InterpreterMode.Expression)
```

Comment:
?

Reply:
`GenerateExprExe()` pertains to the expression interpreter, which was like a watch window for DesignScript debugging during the text IDE days for DS. It's legacy code now, needs to be removed at some point.